operator/hack/common.sh

```diff
@@ -65,6 +65,28 @@ function retry() {
   return 1
 }
 
+function check_version_tag() {
+  local -r version_tag="$1"
+  local -r allow_dirty_tag="$2"
+
+  echo $IMAGE_TAG_BASE
+  if docker manifest inspect "$IMAGE_TAG_BASE:$version_tag" ; then
+    return 0
+  else
+    log "Cannot find $IMAGE_TAG_BASE:$version_tag image tag in the registry."
+  fi
+
+  if [[ "$version_tag" == *-dirty ]]; then
```

Comment:
Note that (unlike with `[ ]`) with `[[ ]]` the quotes around arguments are actually optional even if the variable contains whitespace. I think shellcheck will even complain about them.
cc @SimonBaeumer who I think suggested this

Reply:
Shellcheck did  not complain on my machine. I always add quotes around variables to avoid evaluating variables as shell commands.

Reply:
Shellcheck does not complain for me neither. Although there is such rule [SC2053](https://github.com/koalaman/shellcheck/wiki/SC2053)  but it's not applicable here

Reply:
Looks like I remembered this wrong, don't worry about this.